python/mxnet/image.py

```diff
@@ -60,7 +60,33 @@ def imdecode(buf, **kwargs):
 
 
 def scale_down(src_size, size):
-    """Scale down crop size if it's bigger than image size."""
+    """Scale down crop size if it's larger than image size.
+
+    If width/height of the crop is larger than the width/height of the image,
+    set the width/height to the width/height of the image.
+
+    First compare and set the height, then compare and set the width.
```

Comment:
This looks like an implementation detail and should be omitted from API level documentation.